discovery/manager.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/prometheus/discovery/cvm"
```

Comment:
it is not the correct location for importing this

Reply:
fixed